src/config/config.cc

```diff
@@ -165,6 +165,7 @@ Config::Config() {
       {"unixsocketperm", true, new OctalField(&unixsocketperm, 0777, 1, INT_MAX)},
       {"log-retention-days", false, new IntField(&log_retention_days, -1, -1, INT_MAX)},
       {"persist-cluster-nodes-enabled", false, new YesNoField(&persist_cluster_nodes_enabled, true)},
+      {"redis-cursor-compatible", false, new YesNoField(&redis_cursor_compatible, true)},
```

Comment:
The default value should be false to avoid breaking the old behavior.

Reply:
Wow, I got it. We don't need to make the default configuration file consistent with the default values.

Reply:
Yes, it will break the old users if they upgraded the Kvrocks without adding the new configuration.